abstractions/python/kiota/abstractions/request_information.py

```diff
@@ -79,6 +82,25 @@ def set_url(self, url: Url) -> None:
         self.query_parameters.clear()
         self.path_parameters.clear()
 
+    def get_request_headers(self) -> Optional[Dict]:
+        return self.headers
+
+    def add_request_headers(self, headers_to_add: Optional[Dict[str, str]]) -> None:
+        """Adds headers to the request
+        """
+        if headers_to_add:
+            for key in headers_to_add:
+                self.headers[key.lower()] = headers_to_add[key]
+
+    def remove_request_headers(self, key: str) -> None:
+        """Removes a request header from the current request
+
+        Args:
+            key (str): The key of the header to remove
+        """
+        if key and key.lower() in self.headers:
```

Comment:
Should this be?

```suggestion
        if key and key.lower() in self.headers.keys():
```

Reply:
`.keys()` is not necessary. By default python will iterate over dictionary keys unless you explicitly specify `dict.values()`